parsl/app/errors.py

```diff
@@ -64,10 +64,9 @@ class MissingOutputs(ParslError):
 
     Contains:
     reason(string)
-    outputs(List of strings/files..)
+    outputs(List of files)
```

Comment:
I suppose the typing will make it plain, but might consider `Files` or `List of File objects`